exercises/sum-of-multiples/description.md

```diff
@@ -1,6 +1,9 @@
 # Description
 
-Given a number, find the sum of all the unique multiples of particular numbers up to but not including that number.
+Given a limit and a list of factors, find the sum of all unique multiples of the factors that are less than the limit.
```

Comment:
I completely agree that the original isn't very clear.

I think reversing `limit` and `list of factors` helps with flow, and maybe instead of `find the sum` we can go use the more straight-forward `add up...`.

That gives us:

> Given a list of factors and a limit, add up all the unique multiples of the factors that are less than the limit.

Reply:
The main thing that's still weird about it is that it's ambiguous whether "that are less than the limit" applies to the factors or the unique multiples. But I couldn't work out a nice way to rewrite it (and in any case the example clears it up).

Reply:
Yeah, the example helps!